app/subscriber/src/components/bar/styled/Bar.tsx

```diff
@@ -1,16 +1,18 @@
 import styled from 'styled-components';
 
-export const Bar = styled.div`
+export const Bar = styled.div<{ $vanilla?: boolean }>`
```

Comment:
What's with the use of the `$`?

Reply:
Just from the styled-components docs, seems to be their convention, not a big deal to remove though